Web/starter_project_barbies/components/about/ProjectsList.tsx

```diff
@@ -0,0 +1,71 @@
+import projects from '@/data/about/projects.json'
+import { Project } from '@/types/about'
+import Image from 'next/image'
+import { IconContext } from 'react-icons'
+import { FiGithub } from 'react-icons/fi'
+import { IoMdOpen } from 'react-icons/io'
+
+interface ProjectCardProps {
+  project: Project
+  reverseFlex: boolean
+}
+const ProjectCard: React.FC<ProjectCardProps> = ({ project:{image, name, description}, reverseFlex }) => {
+  return (
+    <div
+      className={`flex flex-wrap justify-around ${
+        reverseFlex ? 'flex-row-reverse' : ''
+      }`}
+    >
+      <Image
+        className="flex-initial"
+        src={`/img/about/projects/${image}`}
+        style={{width:'auto', height:'auto'}}
+        width={700}
+        height={200}
+        alt=""
+      />
+      <div
+        className={`mx-5 ${!reverseFlex ? 'lg:text-right' : 'lg:text-left'}`}
+      >
+        <div className="text-primary">Social Project</div>
+        <div className="text-primary font-bold text-3xl mt-2 mb-5">
+          {name}
+        </div>
+        <div className="lg:max-w-sm text-gray-600">{description}</div>
+        <div
+          className={`flex gap-5 mt-5 ${
+            !reverseFlex ? 'lg:justify-end' : 'lg:justify-start'
```

Comment:
[nit] this is fine for now but generally it is better to not have the negative in the conditional, I'd rather you swap the justify start and justify end instead of adding a not on the reverseFlex
I think you've done that other places too